### ISSUE TITLE
mix.gen.live generated code will error

### ISSUE THREAD

**Issue by bcardarella** [liveview-native/live_view_native#100]
Steps to reproduce:

1. Any current LVN enabled app
2. `mix phx.gen.live Accounts User users name:string`
3. `mix phx.server`

Errors:

```
error: undefined function header/1 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:1: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:5: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function button/1 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:6: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function list/1 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:11: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function back/1 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:15: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:15: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function modal/1 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:17: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:17: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Show.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/show.html.heex:24: LvnDemo3Web.UserLive.Show.LiveViewNative.HTML.render/1

error: undefined function header/1 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:1: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:4: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function button/1 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:5: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function table/1 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:10: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:13: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:18: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:20: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function hide/2 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:24: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function modal/1 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:32: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:32: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1

error: undefined function sigil_p/2 (expected LvnDemo3Web.UserLive.Index.LiveViewNative.HTML to define such a function or for it to be imported, but none are available)
  lib/lvn_demo3_web/live/user_live/index.html.heex:39: LvnDemo3Web.UserLive.Index.LiveViewNative.HTML.render/1


== Compilation error in file lib/lvn_demo3_web/live/user_live/show.ex ==
** (CompileError) lib/lvn_demo3_web/live/user_live/show.ex: cannot compile module LvnDemo3Web.UserLive.Show.LiveViewNative.HTML (errors have been logged)
```

I'm not sure why these modules are being called here, they're being generated by this lib but this template should be rendered by `LvnDemo3Web.UserLive.Index` and not the generated module.

**Comment by supernintendo** [comment]
@bcardarella Fixed by https://github.com/liveview-native/live_view_native/pull/101. I think the HTML platform is a bit of architectural debt that should probably be removed in a future version of `live_view_native`, it doesn't make sense for LVN to be concerned with that.

**Comment by bcardarella** [comment]
Fixed with #101 & #104
